### PR TITLE
Limit sdk based on api config

### PR DESCRIPTION
**Pull request by antho1404**
### Project organization

- Dependant on https://github.com/liteflow-labs/libraries/pull/64
- Dependant on https://github.com/liteflow-labs/backend/pull/2239

### Description

Update hooks to raise an error if the API does not has the feature enabled rather than calling the API and failing anyway

### How to test

Run the API without the plugins you want to test and test that the hooks fail correctly

### Checklist

- [x] Update related changelogs 
- [x] Update docs if necessary

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 #### Breaking Changes
 
+- `TokenFormCreate` does not require the `activateLazyMint` props anymore [](https://github.com/liteflow-labs/libraries/pull/)
```
Comment (antho1404) [inline packages/components/CHANGELOG.md]:
```suggestion
- `TokenFormCreate` does not require the `activateLazyMint` props anymore [#65](https://github.com/liteflow-labs/libraries/pull/65)
```

File: packages/hooks/CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,25 @@
 
 #### Added
 
+- Add `useConfig` hook to return the configuration of the platform [](https://github.com/liteflow-labs/libraries/pull/)
```
Comment (antho1404) [inline packages/hooks/CHANGELOG.md]:
```suggestion
- Add `useConfig` hook to return the configuration of the platform [#65](https://github.com/liteflow-labs/libraries/pull/65)
```

File: packages/hooks/CHANGELOG.md [hunk 3]
```diff
@@ -6,14 +6,25 @@
...
 
+- Use of ts-invariant to assert conditions in the codebase + remove useless checks [#](https://github.com/liteflow-labs/libraries/pull/)
+
 #### Deprecated
 
 #### Removed
 
 #### Fixed
 
+- Add proper errors when calling a hook relying on a feature not activated [](https://github.com/liteflow-labs/libraries/pull/)
```
Comment (antho1404) [inline packages/hooks/CHANGELOG.md]:
```suggestion
- Add proper errors when calling a hook relying on a feature not activated [#65](https://github.com/liteflow-labs/libraries/pull/65)
```

**Review by cpvdeveloper — APPROVED** [review]

**Comment by cpvdeveloper** [comment]
Good idea 👍

**Pull request merged**